app/controllers/vehicle_usages_controller.rb

```diff
@@ -136,6 +138,7 @@ def vehicle_usage_params
       parameters[:vehicle_attributes] = parameters[:vehicle]
       parameters[:vehicle_attributes][:devices] = {} unless parameters[:vehicle_attributes].key?(:devices)
       parameters[:vehicle_attributes][:max_distance] = DistanceUnits.distance_to_meters(parameters[:vehicle_attributes][:max_distance], @current_user.prefered_unit) if parameters[:vehicle_attributes].key?(:max_distance)
+      parameters[:vehicle_attributes][:max_ride_distance] = DistanceUnits.distance_to_meters(parameters[:vehicle_attributes][:max_ride_distance], @current_user.prefered_unit) if parameters[:vehicle_attributes].key?(:max_ride_distance)
```

Comment:
and max_ride_duration ?

Reply:
max_ride_duration is a duration, it does not require a conversion to the prefered_unit which only apply on distances